assets/app/view/game/emergency_money.rb

```diff
@@ -14,13 +14,15 @@ def render_emergency_money_raising(player)
             verticalAlign: 'top',
           },
         }
-        player.shares_by_corporation.each do |corporation, shares|
-          next if shares.empty? || @game.sellable_bundles(player, corporation).empty?
+        if @game.round.actions_for(entity).include?('sell_shares')
```

Comment:
why is this needed?

Reply:
1877 Stockholm Tramways does allow the president to contribute but not by selling shares. Without this the sell shares actions were visible (but not possible to perform because they are not legal actions).